zellij-utils/src/setup.rs

```diff
@@ -444,8 +444,12 @@ impl Setup {
             }
         }
         if let Some(config_file) = config_file {
-            writeln!(&mut message, "[CONFIG FILE]: {:?}", config_file).unwrap();
-            // match Config::new(&config_file) {
+            writeln!(
+                &mut message,
+                "[LOOKING FOR CONFIG FILE FROM]: {:?}",
+                config_file
+            )
+            .unwrap();
             match Config::from_path(&config_file, None) {
                 Ok(_) => message.push_str("[CONFIG FILE]: Well defined.\n"),
                 Err(e) => writeln!(&mut message, "[CONFIG ERROR]: {}", e).unwrap(),
```

Comment:
How about adding this line here like following?
I think that would reduce a bit more confusion.

```
...
[CONFIG ERROR]: IoError: No such file or directory (os error 2), File: ...
 By default zellij will load default configuration
...
```

Reply:
Good suggestion!